VAMobile/patches/@department-of-veterans-affairs+mobile-component-library+0.27.1.patch

```diff
@@ -0,0 +1,67 @@
+diff --git a/node_modules/@department-of-veterans-affairs/mobile-component-library/src/components/Snackbar/Snackbar.tsx b/node_modules/@department-of-veterans-affairs/mobile-component-library/src/components/Snackbar/Snackbar.tsx
+index 6b2f54d..8931ba9 100644
+--- a/node_modules/@department-of-veterans-affairs/mobile-component-library/src/components/Snackbar/Snackbar.tsx
++++ b/node_modules/@department-of-veterans-affairs/mobile-component-library/src/components/Snackbar/Snackbar.tsx
+@@ -160,6 +160,7 @@ export const Snackbar: FC<SnackbarProps> = (toast) => {
+       minHeight: 44,
+       padding: spacing.vadsSpaceSm,
+       marginHorizontal: spacing.vadsSpaceLg,
++      marginBottom: spacing.vadsSpaceLg,
```

Comment:
I see the justification for this in another comment:
> I made another patch to the Snackbar component to update the navigation bar height (more accurate) and to add a bottom margin to the Snackbar. This seemed like the right place to fix things since we are already accounting for the left/right margins and navigation bar height in the component itself versus relying on the app to provide these properties.

The reason bottom margin was not included in the component is two fold:
1. It's controllable via the offset to whatever an app could want so we just gave a default as apps would need to override it anyway for any special cases--if 20 pixel spacing is desired without a Nav bar, offset can be set to 20
2. Margin makes elements underneath the Snackbar (e.g. a button sticking out partially) no longer pressable within the margined space despite being visible where setting the offset via the library used by the component allows presses outside the bounds of the Snackbar to still occur as desired
   - For full disclosure: when building it, I tried to use margin instead of the library offset functionality since it was simpler, but found it resulted in the bug where you could position it above the Nav bar visually, but none of the Nav bar buttons would be able to be pressed; using the offset functionality, the Nav bar can still be interacted with as meets user expectations

The side margins were deemed acceptable to block presses as, paired with general padding for content, anything peeking should be very minimal.

I technically defer to the app team since we on the design system team long term are not responsible for how apps implement our components, but I'd recommend against patching this in as patches are frail and this is liable to break from future library changes we make to the component when updating while leveraging the offset should not.

Reply:
Thanks for the feedback. Just so we're all on the same page, the left screenshot below is the default location where the Snackbar gets positioned when a bottom navigation bar is present and no offset is specified. There are 4 units of vertical spacing between the top of the navbar and the Snackbar. The right screenshot shows the changes with the patch applied (20 units of vertical spacing). If we can live with the spacing on the left screenshot, then I can remove the patch easily enough. If we want more spacing but no patch, then the app will have to start passing offsets into all Snackbars. Please advise.

<div style="display: flex; justify-content: space-around;">
  <img src="https://github.com/user-attachments/assets/f62a05c8-2ad6-4f4a-ae2a-b6f8f2511b46" width="200"/>
  <img src="https://github.com/user-attachments/assets/817a8379-8a12-46f9-9d46-5c635ae82f68" width="200"/>
</div>

Reply:
I think the 4 units spacing is fine, but ultimately seems like a design decision. I believe the position is what it was with the old Snackbar so it'd be maintaining existing functionality and didn't come up during creating it or we'd likely have gone with a different default. As a "floating" element it can appear directly above other content regardless if it's 4 or 20 pixels from the Nav bar--the crux is not blocking the Nav bar so it's usable.

If the increased spacing is determined the way: I think a patch is fair way to go about it, but it would be better to update the default offset to 76 instead of using `marginBottom`. Longer term, we have done research into global overrides for certain design system parameters and the Snackbar offset definitely would be a good use case--but, in the absence of that yet existing, patching in a different default value seems like a reasonable workaround since when the global override exists then the only code change would be removing the patch and setting the global override in one spot.

Not sure if @Sparowhawk or anyone else on the app side had thoughts since ultimately from the design system side we're just providing recommendations based on how the components were built/designed and consumer apps can do what they want.

Reply:
As a circle back: [created a ticket](https://github.com/department-of-veterans-affairs/va-mobile-library/issues/609) to add a global override for the default Snackbar offset so it's documented as a desired future enhancement.